conf/squid/common/13node-4client-single-site-regression.yaml

```diff
@@ -0,0 +1,120 @@
+# Single site cluster with 13 Ceph & 4 Client nodes
+# with daemons - 13 crash, 5 mon, 3 mgr, 3 rgw
+# 5 nfs, 10 (+2 backup) osds and
+# 1 alertmanager, node-exporter, prometheus, grafana
+globals:
+  - ceph-cluster:
+      name: ceph
+      node1:
+        role:
+          - _admin
+          - installer
+          - crash
+          - alertmanager
+          - mon
+      node2:
+        role:
+          - crash
+          - rgw
+          - nfs
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node3:
+        role:
+          - crash
+          - rgw
+          - nfs
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node4:
+        role:
+          - crash
+          - rgw
+          - nfs
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node5:
+        role:
+          - crash
+          - nfs
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node6:
+        role:
+          - crash
+          - nfs
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node7:
+        role:
+          - crash
+          - node-exporter
+          - mds
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node8:
+        role:
+          - crash
+          - prometheus
+          - mds
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node9:
+        role:
+          - crash
+          - grafana
+          - mds
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node10:
+        role:
+          - crash
+          - mon
+          - mds
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node11:
+        role:
+          - crash
+          - mon
+          - mds
+          - osd
+        no-of-volumes: 4
+        disk-size: 20
+      node12:
+        role:
+          - crash
+          - mon
+          - mgr
+          - osd-bak
+        no-of-volumes: 4
+        disk-size: 20
+      node13:
+        role:
+          - crash
+          - mon
+          - mgr
+          - osd-bak
```

Comment:
what is this `osd-bak` like that role exist from cephadm side.

Reply:
this is for RADOS stretch cluster [1], I am assuming these are used as backup osd nodes.
[1] https://github.com/red-hat-storage/cephci/blob/main/conf/squid/rados/7node-1client-stretch-mode-rhos-d.yaml